x/oracle/types/prophecy.go

```diff
@@ -93,10 +94,28 @@ func (prophecy Prophecy) AddClaim(validator sdk.ValAddress, claim string) {
 	prophecy.ValidatorClaims[validatorBech32] = claim
 }
 
+func inWhiteList(validator staking.Validator, whiteListValidatorAddresses []sdk.ValAddress) bool {
+	for _, whiteListValidatorAddress := range whiteListValidatorAddresses {
+		if bytes.Equal(validator.GetOperator(), whiteListValidatorAddress) {
+			return true
+		}
+	}
+	return false
+}
+
 // FindHighestClaim looks through all the existing claims on a given prophecy. It adds up the total power across
 // all claims and returns the highest claim, power for that claim, and total power claimed on the prophecy overall.
-func (prophecy Prophecy) FindHighestClaim(ctx sdk.Context, stakeKeeper StakingKeeper) (string, int64, int64) {
+func (prophecy Prophecy) FindHighestClaim(ctx sdk.Context, stakeKeeper StakingKeeper, whiteListValidatorAddresses []sdk.ValAddress) (string, int64, int64, int64) {
```

Comment:
I think the function comment above needs to be updated to talk about whitelisting

Reply:
comments updated in new commit.